packages/camera/camera_avfoundation/ios/Classes/FLTCamMediaSettings.m

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTCamMediaSettings.h"
+
+@implementation FLTCamMediaSettings
+
+NS_INLINE void AssertPositiveNumberOrNil(NSNumber *_Nullable param,
```

Comment:
This should be `static`. (It's also not clear to me why it's `NS_INLINE`; do you have reason to believe the method call would be a performance problem?)

Reply:
remove inline, changed to static